doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -439,10 +441,11 @@ The current implementation recognizes the following built-in quotations:
   holes at runtime (type ``Init.constr`` as well).
 - ``pattern``, which parses Coq patterns and produces a pattern used for term
   matching (type ``Init.pattern``).
-- ``reference``, which parses either a :n:`@qualid` or :n:`& @ident`. Qualified names
+- ``reference``, which parses either a :n:`@qualid` or :n:`&@ident`. Qualified names
   are globalized at internalization into the corresponding global reference,
   while ``&id`` is turned into ``Std.VarRef id``. This produces at runtime a
-  ``Std.reference``.
+  ``Std.reference``. There shall be no white space between the ampersand
+  symbol (``&``) and the identifier (:n:`@ident`).
```

Comment:
What about explaining the reason in passing.

Reply:
I'm not sure we should document all the limitations that are behind choices of syntax, otherwise it is going to make relevant information harder to find.

Reply:
Dunno, the only thing that I would write here is "to resolve ambiguity with existing notations", that given we have an extensible grammar is not particularly interesting, I believe.